src/Cpp/1-getting-started/1-HelloWindow/Input/Input.cpp

```diff
@@ -0,0 +1,56 @@
+#include "Input.hpp"
+
+#include <GLFW/glfw3.h>
+
+#include <functional>
```

Comment:
what are we including `<functional>` for?

Reply:
Ah that was a leftover from me dinglefarting around

Reply:
Ok thats why github reviews SUCK, because reviewed code doesnt update accordingly ;) like in azure dev ops

I have removed <functional>